index.js

```diff
@@ -1,69 +1,20 @@
 const core = require("@actions/core");
-const github = require("@actions/github");
-const fs = require("fs");
+const exec = require("@actions/exec");
 
 async function run() {
   try {
-    const token = core.getInput("github-token");
     const pizzaArgs = core.getInput("pizza-args");
+    const args = pizzaArgs.split(/\s+/).filter((arg) => arg.length > 0);
 
-    // Run the pizza CLI command using the given pizza arguments
-    const { execSync } = require("child_process");
-    execSync(`pizza ${pizzaCommand}`);
+    console.log("Running Pizza CLI with args:", args);
+    await exec.exec("pizza", args);
```

Comment:
I've tried:
```javascript
await exec.exec("pizza", args);
```

```javascript
await exec.exec(`pizza ${pizzaArgs}`);
```
and the equivalents with `execSync` etc. but they all result in running the pizza CLI without the args and the args are present in `pizzaArgs`. I'm sure it's something super silly too.

```
Run node /home/runner/work/_actions/open-sauced/pizza-action/nickytonline/implement-action/index.js
Running Pizza CLI with args: [ 'generate', 'codeowners', './' ]
/opt/hostedtoolcache/node/20.17.0/x64/bin/pizza generate codeowners ./
A command line utility for insights, metrics, and all things OpenSauced
Usage:
  pizza <command> <subcommand> [flags]
Available Commands:
  bake        Use a pizza-oven to source git commits into OpenSauced
  completion  Generate the autocompletion script for the specified shell
  help        Help about any command
  insights    Gather insights about git contributors, repositories, users and pull requests
  login       Log into the CLI application via GitHub
  repo-query  Ask questions about a GitHub repository
  show        Get visual metrics of a repository
  version     Displays the build version of the CLI
Flags:
      --beta                Shorthand for using the beta OpenSauced API
                            endpoint
                            ("https://beta.api.opensauced.pizza/").
                            Supersedes the '--endpoint' flag
      --disable-telemetry   Disable sending telemetry data to OpenSauced
  -e, --endpoint string     The API endpoint to send requests to (default
                            "https://api.opensauced.pizza/")
  -h, --help                help for pizza
  -o, --output string       The formatting for command output. One of:
                            (table, yaml, csv, json) (default "table")
Use "pizza [command] --help" for more information about a command.
/usr/bin/git config --global user.name github-actions
/usr/bin/git config --global user.email github-actions@github.com
/usr/bin/git add .
/usr/bin/git commit -m Auto-commit from OpenSauced Pizza Action
On branch main
Your branch is up to date with 'origin/main'.
nothing to commit, working tree clean
Error: The process '/usr/bin/git' failed with exit code 1
Error: Process completed with exit code 1.
```

Reply:
I'll be deleting this test repo once this is merged, but you can see the different tries. 😅

Reply:
I'm going to try running it from a bash script.

Reply:
Being explicit about the encoding changes nothing for the args not being passed in. Going to try the bash script on the fly option.

Reply:
I think I figured it out while trying the bash way. I need to strip the quotes.

```
pizza "generate" "codeowners" "./"
Script path: /home/runner/work/test-action/test-action/run_pizza.sh
/usr/bin/bash /home/runner/work/test-action/test-action/run_pizza.sh
A command line utility for insights, metrics, and all things OpenSauced
Usage:
  pizza <command> <subcommand> [flags]
Available Commands:
  bake        Use a pizza-oven to source git commits into OpenSauced
  completion  Generate the autocompletion script for the specified shell
  help        Help about any command
  insights    Gather insights about git contributors, repositories, users and pull requests
  login       Log into the CLI application via GitHub
  repo-query  Ask questions about a GitHub repository
  show        Get visual metrics of a repository
  version     Displays the build version of the CLI
Flags:
      --beta                Shorthand for using the beta OpenSauced API
                            endpoint
                            ("https://beta.api.opensauced.pizza/").
                            Supersedes the '--endpoint' flag
      --disable-telemetry   Disable sending telemetry data to OpenSauced
  -e, --endpoint string     The API endpoint to send requests to (default
                            "https://api.opensauced.pizza/")
  -h, --help                help for pizza
  -o, --output string       The formatting for command output. One of:
                            (table, yaml, csv, json) (default "table")
```

Reply:
Hmm, still no dice.

```
Raw pizza-args: generate codeowners ./
Stripped args: [ 'generate', 'codeowners', './' ]
Bash script content: #!/bin/bash
set -e
pizza generate codeowners ./
Executing bash script...
/usr/bin/bash /home/runner/work/test-action/test-action/run_pizza.sh
A command line utility for insights, metrics, and all things OpenSauced
Usage:
  pizza <command> <subcommand> [flags]
Available Commands:
  bake        Use a pizza-oven to source git commits into OpenSauced
  completion  Generate the autocompletion script for the specified shell
  help        Help about any command
  insights    Gather insights about git contributors, repositories, users and pull requests
  login       Log into the CLI application via GitHub
  repo-query  Ask questions about a GitHub repository
  show        Get visual metrics of a repository
  version     Displays the build version of the CLI
Flags:
      --beta                Shorthand for using the beta OpenSauced API
                            endpoint
                            ("https://beta.api.opensauced.pizza/").
                            Supersedes the '--endpoint' flag
      --disable-telemetry   Disable sending telemetry data to OpenSauced
  -e, --endpoint string     The API endpoint to send requests to (default
                            "https://api.opensauced.pizza/")
  -h, --help                help for pizza
  -o, --output string       The formatting for command output. One of:
                            (table, yaml, csv, json) (default "table")
Use "pizza [command] --help" for more information about a command.
/usr/bin/git config --global user.name github-actions
/usr/bin/git config --global user.email github-actions@github.com
/usr/bin/git add .
/usr/bin/git commit -m Auto-commit from OpenSauced Pizza Action
```

This looks good though 🤔 

```
Raw pizza-args: generate codeowners ./
Stripped args: [ 'generate', 'codeowners', './' ]
Bash script content: #!/bin/bash
set -e
pizza generate codeowners ./
Executing bash script...
/usr/bin/bash /home/runner/work/test-action/test-action/run_pizza.sh
```

I'll get back on this tomorrow morning.